.circleci/config.yml

```diff
@@ -25,7 +30,7 @@ jobs:
           when: on_success
           paths:
             - ~/.local/share/virtualenvs/
```

Comment:
When we save the cache, we want to cache

- `./.venv`
- `$HOME/.cache/uv`

`~/.local/share/virtualenvs/` shouldn't matter any more

Reply:
https://github.com/DemocracyClub/ec-postcode-lookup-pages/pull/117/commits/7cab83a01f204b6852297e55d2bc3901ee232f14